runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/commits/StreamingEngineWorkCommitter.java

```diff
@@ -174,19 +175,18 @@ private void streamingCommitLoop() {
           continue;
         }
 
-        try (CloseableStream<CommitWorkStream> closeableCommitStream =
-                commitWorkStreamFactory.get();
-            CommitWorkStream.RequestBatcher batcher = closeableCommitStream.stream().batcher()) {
+        try (CommitWorkStream.RequestBatcher batcher = closeableCommitStream.stream().batcher()) {
           if (!tryAddToCommitBatch(initialCommit, batcher)) {
             throw new AssertionError("Initial commit on flushed stream should always be accepted.");
           }
           // Batch additional commits to the stream and possibly make an un-batched commit the
           // next initial commit.
           initialCommit = expandBatch(batcher);
-        } catch (Exception e) {
-          LOG.error("Error occurred sending commits.", e);
```

Comment:
stream timeouts were getting caught here, now they will not be caught. Thinking if we should handle it differently, is it safe to suppress these exceptions?